templates/base/src/pages/_app.js

```diff
@@ -18,7 +18,9 @@ function MyApp({ Component, pageProps }) {
     <ApolloProvider client={GraphQLClient}>
       <ThemeProvider theme={theme}>
         <GlobalContextProvider>
-          <Component {...pageProps} />
+          <ReactQueryProvider>
```

Comment:
I am thinking to make this an optional feature. I need to think more on this, will get back to you on this one. @SanjanaPalisetti